test/msw-api/unregister.test.ts

```diff
@@ -35,10 +35,7 @@ test('unregisters itself when not prompted to be activated again', async () => {
   })
   const secondBody = await secondResponse.json()
 
-  // Although the Service Worker unregisters itself upon refreshing the page,
-  // it still remains "active and running" with the "deleted" status.
-  // This results into requests go through the Service Worker until the next reload.
-  expect(secondResponse.fromServiceWorker()).toBe(true)
+  expect(secondResponse.fromServiceWorker()).toBe(false)
```

Comment:
Moving the navigation request bypassing out of the `respondWith` scope now fixes the issue that marked the first request as served from the worker, even after the worker has been unregistered. This test is now actually correct.